<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"><class>MainWindow</class><widget class="QMainWindow" name="MainWindow"><property name="geometry"><rect><x>0</x><y>0</y><width>390</width><height>370</height></rect></property><property name="sizePolicy"><sizepolicy hsizetype="Expanding" vsizetype="Expanding"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy></property><property name="minimumSize"><size><width>390</width><height>370</height></size></property><property name="maximumSize"><size><width>390</width><height>370</height></size></property><property name="windowTitle"><string>Моделирование</string></property><widget class="QWidget" name="centralWidget"><widget class="QListWidget" name="resultWidget"><property name="geometry"><rect><x>10</x><y>125</y><width>370</width><height>220</height></rect></property><property name="editTriggers"><set>QAbstractItemView::NoEditTriggers</set></property><property name="defaultDropAction"><enum>Qt::IgnoreAction</enum></property></widget><widget class="QLineEdit" name="referLine"><property name="geometry"><rect><x>200</x><y>10</y><width>180</width><height>27</height></rect></property><property name="text"><string>1 2 3 1 2 3 1 2 3 4 5</string></property></widget><widget class="QLabel" name="label"><property name="geometry"><rect><x>10</x><y>13</y><width>180</width><height>20</height></rect></property><property name="text"><string>Ссылочная строка:</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget><widget class="QRadioButton" name="fifoButton"><property name="geometry"><rect><x>10</x><y>70</y><width>70</width><height>25</height></rect></property><property name="text"><string>FIFO</string></property><property name="checked"><bool>true</bool></property></widget><widget class="QRadioButton" name="reverseLRUButton"><property name="geometry"><rect><x>80</x><y>70</y><width>121</width><height>25</height></rect></property><property name="text"><string>ReverseLRU</string></property></widget><widget class="QPushButton" name="startButton"><property name="geometry"><rect><x>200</x><y>70</y><width>180</width><height>25</height></rect></property><property name="text"><string>Начать</string></property></widget><widget class="QLabel" name="label_2"><property name="geometry"><rect><x>10</x><y>43</y><width>180</width><height>20</height></rect></property><property name="text"><string>Число страниц:</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget><widget class="QSpinBox" name="frameCountSpin"><property name="geometry"><rect><x>200</x><y>40</y><width>180</width><height>27</height></rect></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="minimum"><number>1</number></property><property name="maximum"><number>1000</number></property><property name="singleStep"><number>1</number></property><property name="value"><number>3</number></property></widget><widget class="QRadioButton" name="clockButton"><property name="geometry"><rect><x>10</x><y>95</y><width>70</width><height>25</height></rect></property><property name="text"><string>Clock</string></property><property name="checked"><bool>false</bool></property></widget><widget class="QRadioButton" name="lfuButton"><property name="geometry"><rect><x>140</x><y>95</y><width>60</width><height>25</height></rect></property><property name="text"><string>LFU</string></property></widget><widget class="QRadioButton" name="lruButton"><property name="geometry"><rect><x>80</x><y>95</y><width>60</width><height>25</height></rect></property><property name="text"><string>LRU</string></property></widget><widget class="QPushButton" name="infoButton"><property name="geometry"><rect><x>200</x><y>95</y><width>180</width><height>25</height></rect></property><property name="text"><string>Информация</string></property></widget></widget><widget class="QStatusBar" name="statusBar"/></widget><layoutdefault spacing="6" margin="11"/><resources/><connections/></ui>
